code/game/dna/genes/vg_powers.dm

```diff
@@ -37,70 +37,101 @@
 		to_chat(usr, "\red You can't change your appearance right now!")
 		return
 	var/mob/living/carbon/human/M=usr
+	var/obj/item/organ/external/head/head_organ = M.get_organ("head")
 
-	var/new_facial = input("Please select facial hair color.", "Character Generation",rgb(M.r_facial,M.g_facial,M.b_facial)) as null|color
-	if(new_facial)
-		M.r_facial = hex2num(copytext(new_facial, 2, 4))
-		M.g_facial = hex2num(copytext(new_facial, 4, 6))
-		M.b_facial = hex2num(copytext(new_facial, 6, 8))
-
-	var/new_hair = input("Please select hair color.", "Character Generation",rgb(M.r_hair,M.g_hair,M.b_hair)) as null|color
-	if(new_facial)
-		M.r_hair = hex2num(copytext(new_hair, 2, 4))
-		M.g_hair = hex2num(copytext(new_hair, 4, 6))
-		M.b_hair = hex2num(copytext(new_hair, 6, 8))
+	var/new_gender = alert(usr, "Please select gender.", "Character Generation", "Male", "Female")
```

Comment:
Moved Gender up top so if someone wants to turn into a male and have a beard, they can do so in one cast of morph.